src/nvim/option_defs.h

```diff
@@ -473,6 +473,12 @@ EXTERN char_u   *p_isf;         // 'isfname'
 EXTERN char_u   *p_isi;         // 'isident'
 EXTERN char_u   *p_isp;         // 'isprint'
 EXTERN int p_js;                // 'joinspaces'
+EXTERN char_u *p_jop;           // 'jumpooptions'
+EXTERN unsigned jop_flags;
+# ifdef IN_OPTION_C
```

Comment:
```suggestion
#ifdef IN_OPTION_C
```

Reply:
Fixed in https://github.com/neovim/neovim/pull/11530/commits/3aca2ea5bcaf9cbc1ad7eb93ca98176e753cbfd4 .